1832.check-if-the-sentence-is-pangram.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @param {string} sentence
+ * @return {boolean}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/check-if-the-sentence-is-pangram/
+*/
+
+ var checkIfPangram = function(sentence) {
+    const alphabetTable = {};
+    
+    for (let i = 0; i < sentence.length; i++) {
+        if(alphabetTable[sentence[i]] === undefined) {
+            alphabetTable[sentence[i]] = 1;
+        }
+    }
+    
+    return Object.keys(alphabetTable).length === 26 ? true : false; 
```

Comment:
```suggestion
    return Object.keys(alphabetTable).length === 26
```
this works the same.

Reply:
I removed the last part starting with ?.